frontend/src/citizen-frontend/calendar/CalendarPage.tsx

```diff
@@ -89,6 +97,13 @@ export default React.memo(function CalendarPage() {
     [data]
   )
 
+  const isHolidayFormActive: boolean = holidayPeriods.mapAll({
+    loading: () => false,
+    failure: () => false,
+    success: ([holidayPeriods]) =>
+      holidayPeriods != undefined && holidayPeriods != null
```

Comment:
Param naming is slightly confusing since the destuctured item is a single period.
```suggestion
    success: (periods) => periods.length > 0
```
🤷 Could use memo here too.